grafana-plugin/src/containers/DefaultPageLayout/DefaultPageLayout.tsx

```diff
@@ -79,7 +79,7 @@ const DefaultPageLayout: FC<DefaultPageLayoutProps> = observer((props) => {
             // @ts-ignore
             title="Slack integration warning"
           >
-            {getSlackMessage(showSlackInstallAlert, store.teamStore.currentTeam)}
+            {getSlackMessage(showSlackInstallAlert, store.teamStore.currentTeam, store)}
```

Comment:
The store is used just for `store.hasFeature(AppFeature.LiveSettings)`, you could pass that instead as `hasLiveSettingsFeature`